frontend/src/Components/common/Popover.tsx

```diff
@@ -48,20 +26,18 @@ const PopoverMenu = ({ open, anchorEl, handleClosePopover }: PopoverMenuProps) =
     >
       <List>
         <ListItem sx={{p: 0}}>
-          <ListItemButton onClick={handleOpenRenameChannel}>
+        <ListItemButton onClick={() => handleClosePopover('openRename')}>
```

Comment:
```suggestion
        <ListItemButton onClick={() => handleClosePopover(PopoverAction.Rename)}>
```

минорно,но так меньше вероятности допустить опечатку и более надежно